platform/icon/api_test.go

```diff
@@ -0,0 +1,66 @@
+package icon
+
+import(
+	"github.com/trustwallet/blockatlas/models"
+	"github.com/trustwallet/blockatlas/coin"
+	"encoding/json"
+	"testing"
+	"bytes"
+)
+
+const basicSrc = `
+{
+	"txHash": "0x34b8b6ec3a52710c24074f5e298f4a9c67bb61a0a1dde20e695efaeb30ff3754",
+	"height": 357832,
+	"createDate": "2019-04-16T06:36:34.000+0000",
+	"fromAddr": "hx1b8959dd5c57d2c502e22ee0a887d33baec09091",
+	"toAddr": "cx334db6519871cb2bfd154cec0905ced4ea142de1",
+	"txType": "1",
+	"dataType": "call",
+	"amount": "0",
```

Comment:
use example with amount > 0

Reply:
Done in https://github.com/TrustWallet/blockatlas/pull/67/commits/e84c7b43ecbfa0f5a558e2d36a718b07ea47f70a